### ISSUE TITLE
Informatie model sensor registry uitwerken

### ISSUE THREAD

**Issue by marcvanandel** [kadaster-labs/sensrnet-home#28]
Voor het uitwerken van een officieel informatie model is het nodig om ook de taxonomie rondom sensoren duidelijk te maken. Daarvoor kan gebruik gemaakt worden van [SKOS](https://en.wikipedia.org/wiki/Simple_Knowledge_Organization_System) en hoe daarvoor een '[legal extension](https://github.com/bp4mc2/bp4mc2/blob/master/profiles/skoslex-ap-sc.ttl)' is ontwikkeld. (zie ook [Zorgeloos Vastgoed](http://taxonomie.zorgeloosvastgoed.nl/) ([sources](https://github.com/bp4mc2/bp4mc2-zvg))

**Comment by kad-bloemy** [comment]
@marcvanandel Als dit al moet gebeuren, dan zeker niet voor het MVP, dat nu kennelijk wel als milestone staat. Bestaat er ook nog een verdere future? ;)

**Comment by marcvanandel** [comment]
Ja, goeie. Ik denk dat het officieel vaststellen en publiceren van het datamodel 'out-of-scope van MVP' is ... maar wellicht ontstaat dat nog wel eens. Open laten voor 'future'?

**Comment by kad-bloemy** [comment]
Staat ie al op, prima.
